.github/workflows/release.yml

```diff
@@ -11,7 +11,7 @@ jobs:
     runs-on: ubuntu-18.04
     steps:
     - name: Checkout
-      uses: actions/checkout@v2
+      uses: actions/checkout@v2.3.4
```

Comment:
```suggestion
      uses: actions/checkout@v2.3.4
      with:
        fetch-depth: 0
```

Reply:
We have infrequent releases, but it seems reasonable to add this now and see how it works next time we do a release.